lib/fluent/output.rb

```diff
@@ -98,7 +98,7 @@ class OutputThread
     def initialize(output)
       @output = output
       @finish = false
-      @next_time = Engine.now + 1.0
+      @next_time = Time.now.to_f + 1.0
```

Comment:
OK.  I now feel that changing to Time objects here is kinda overkill to this original issue.  It is a good thing to have, but we can live without them.